src/test/scala/mesosphere/marathon/api/v2/AppsResourceTest.scala

```diff
@@ -341,6 +342,89 @@ class AppsResourceTest extends MarathonSpec with MarathonActorSupport with Match
     JsonTestHelper.assertThatJsonString(response.getEntity.asInstanceOf[String]).correspondsToJsonOf(expected)
   }
 
+  /*
+  test("Create a new app with IP/CT with virtual network foo w/ Docker") {
```

Comment:
I really want this version of the unit test to work but I get a validation error with ClassCastExceptions that follow. no comprendo.